docs/stylesheets/style.css

```diff
@@ -632,11 +632,13 @@ p {
 @media only screen and (max-width: 768px) {
   .nav__items {
     flex-direction: column;
+    display: flex;
     position: fixed;
-    top: calc(calc(var(--gap-lg) * 2 + 32px) + 0.5rem);
+    top: 0;
     left: 0;
-    width: 100vw;
-    background-color: var(--color-navigation);
+    width: 100%;
+    height: min-content;
+    background-color: var(--color-navigation) !important;
```

Comment:
generally avoid !important in CSS if possible, this causes headaches when trying to debug and figuring out the precedence of the lines

Reply:
Tested and removed - thanks!